src/core/reducer.js

```diff
@@ -336,6 +336,10 @@ export function CreateGameReducer({ game, multiplayer }) {
           return state;
         }
 
+        if (action.payload.playerID && action.payload.playerID !== state.ctx.currentPlayer) {
```

Comment:
This `action.payload.playerID && ` guard clause seems unnecessary because playerID always exists. I prefer to delete this clause by regarding `playerID` as required parameter, but it needs fixing some existing tests, so I added it for now and would like to listen your opinion.

Reply:
Sure, let's assume that it is always sent and fix the tests instead.